unit-tests/py/rspy/devices.py

```diff
@@ -3,6 +3,8 @@
 
 import sys, os, re, platform
 
+from rspy import device_hub
```

Comment:
sure, we can also move it right before `hub = device_hub.create()` because that's the only place that uses the import? inside the try block

also, why would it fail?